todolist/settings.py

```diff
@@ -62,19 +62,14 @@
 # https://docs.djangoproject.com/en/1.7/ref/settings/#databases
 
 DATABASES = {
-    "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": os.path.join(BASE_DIR, "db.sqlite3"),
+    'default': {
+        'ENGINE': 'mysql.connector.django',
+        'NAME': 'app_db',
+        'USER': 'app_user',
+        'PASSWORD': '1234',
+        'HOST': '172.17.0.2',  # You can use a different host in your MySQL server is on a remote machine.
```

Comment:
The HOST is set to a specific IP address. Ensure this is correct for your environment, especially if the MySQL server is running on a different machine or container.

Reply:
It's correct and according to README.md task:

"
8. Update the Python app db config with an IP of a running MySQL server container (without it, the app container won’t build)
"